themes/prism-hopscotch.css

```diff
@@ -9,8 +9,7 @@
 
 code[class*="language-"],
 pre[class*="language-"] {
-	color: #ffffff;
```

Comment:
I'm assuming this is removed cuz it's the default color?

Reply:
It's removed because at the end of rule is overwritten with `#b9b5b8`.

This is one of the "errors" (should I really call this an error? It's just non-functional code.) in the themes. Really hard to spot for humans but easy for linters.